python/pyspark/sql/catalog.py

```diff
@@ -29,31 +31,35 @@
 Column = namedtuple("Column", "name description dataType nullable isPartition isBucket")
 Function = namedtuple("Function", "name description className isTemporary")
 
+if TYPE_CHECKING:
+    from pyspark.sql._typing import UserDefinedFunctionLike
+    from pyspark.sql.types import DataType
+
```

Comment:
nit: shall we move these to between after the other imports and the definition of `Database`?

Reply:
Sounds good, updated!